app/views/manager/candidatures/index.html.erb

```diff
@@ -0,0 +1,60 @@
+                </svg>
+              </div>
+              <%= f.search_field :company_name_cont, class: "bg-gray-50 border border-gray-300 text-gray-900 text-sm rounded-lg focus:ring-primary-500 focus:border-primary-500 block w-full pl-10 p-2 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-primary-500 dark:focus:border-primary-500", placeholder: t('views.manager.goals.index.search') %>
+            </div>
+            <button type="submit" class="bg-blue-700 text-white rounded-md px-3 py-2 ml-2 hover:bg-primary-600 focus:outline-none focus:ring focus:ring-primary-300 dark:bg-primary-600 dark:hover:bg-primary-700 dark:focus:ring-primary-300">
+              <svg aria-hidden="true" class="w-5 h-5 text-white dark:text-white" fill="currentColor" viewbox="0 0 20 20" xmlns="http://www.w3.org/2000/svg">
+                <path fill-rule="evenodd" d="M8 4a4 4 0 100 8 4 4 0 000-8zM2 8a6 6 0 1110.89 3.476l4.817 4.817a1 1 0 01-1.414 1.414l-4.816-4.816A6 6 0 012 8z" clip-rule="evenodd"/>
+              </svg>
+            </button>
+          <% end %>
+        </div>
+        <div class="w-full md:w-auto flex flex-col md:flex-row space-y-2 md:space-y-0 items-stretch md:items-center justify-end md:space-x-3 flex-shrink-0">
+          <%= button_to new_manager_candidature_path, method: :get, class: "flex items-center justify-center text-white bg-blue-700 hover:bg-primary-800 focus:ring-4 focus:ring-primary-300 font-medium rounded-lg text-sm px-4 py-2 dark:bg-primary-600 dark:hover:bg-primary-700 focus:outline-none dark:focus:ring-primary-800" do %>
+            <svg class="h-3.5 w-3.5 mr-2" fill="currentColor" viewbox="0 0 20 20" xmlns="http://www.w3.org/2000/svg" aria-hidden="true">
+              <path clip-rule="evenodd" fill-rule="evenodd" d="M10 3a1 1 0 011 1v5h5a1 1 0 110 2h-5v5a1 1 0 11-2 0v-5H4a1 1 0 110-2h5V4a1 1 0 011-1z"/>
+            </svg>
+            <%= t('manager.candidatures.index.add_candidature') %>
+          <% end %>
+
+        </div>
+      </div>
+      <div class="overflow-x-auto">
+        <table class="w-full text-sm text-left text-gray-500 dark:text-gray-400">
+          <thead class="text-xs text-gray-700 uppercase bg-gray-50 dark:bg-gray-700 dark:text-gray-400">
+          <tr>
+            <th scope="col" class="px-4 py-3"></th>
+            <th scope="col" class="px-4 py-3"><%= t('manager.candidatures.index.company_name') %></th>
+            <th scope="col" class="px-4 py-3">Frame Work</th>
+            <th scope="col" class="px-4 py-3"><%= t('manager.candidatures.index.programming_language') %></th>
+            <th scope="col" class="px-4 py-3"><%= t('manager.candidatures.index.situation') %></th>
+            <th scope="col" class="px-4 py-3">
+              <span class="sr-only"><%= t('views.manager.goals.index.filter_by') %></span>
+            </th>
```

Comment:
Não entendi a finalidade dessa coluna culta.

Acredito que não devemos reutilizar traduções de outras entidades da aplicação. Se for necessário alterar apenas para Goals, Candidatures também seria alterada indesejadamente.

Bom confirmar com Marco, mas lembro que Goals, futuramente será removido.